src/CoreText/CTTypesetter.cs

```diff
@@ -94,6 +71,15 @@ public CTTypesetterOptions (NSDictionary dictionary)
 			get {return Adapter.GetInt32Value (Dictionary, CTTypesetterOptionKey.ForceEmbeddingLevel);}
 			set {Adapter.SetValue (Dictionary, CTTypesetterOptionKey.ForceEmbeddingLevel, value);}
 		}
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		public bool AllowUnboundedLayout {
+			get => CFDictionary.GetBooleanValue (Dictionary.Handle, CTTypesetterOptionKey.AllowUnboundedLayout.Handle);
```

Comment:
that will NRE before the fields are available :|
was the `Adapter.*` code above checking for null values ?

Reply:
Yes, adapter also has this issue, we'll need to audit this even for StrongDictionaries as we discussed, will check and prepare an RFC